File: leetCodePractice/string/validAnagram.js
/**
 * @param {string} s
 * @param {string} t
 * @return {boolean}
 */
 var isAnagram = function(s, t) {
    if (s.length !== t.length) {
        return false;
    }
    for (let i=0; i < s.length; i++) {
        t = t.replace(s[i], "");
    }
    return t.length === 0;
};

File: leetCodePractice/string/validAnagram.py
class Solution(object):
    def isAnagram(self, s, t):
        """
        :type s: str
        :type t: str
        :rtype: bool
        """
        if len(s) != len(t):
            return False
        
        for i in range(len(s)):
            if s[i] in t:   
                t = t.replace(s[i], "", 1)
        
        return len(t) == 0
        
            